docs/wiki/ModulesUsage.md

```diff
@@ -62,7 +72,7 @@ $inputObject = @{
 New-AzResourceGroupDeployment @inputObject
```

Comment:
I know this is an example, but should TemplateUri = ... Azure/ResourceModules ?

Reply:
It should. Update both local & remote